docs/example_compartmental_elongated_cells_tutorial.rst

```diff
@@ -0,0 +1,805 @@
+            <Lambda>100.0</Lambda>
+            <ActivationEnergy>-50.0</ActivationEnergy>
+            <TargetDistance>5</TargetDistance>
+            <MaxDistance>10.0</MaxDistance>
+            <MaxNumberOfJunctions>2</MaxNumberOfJunctions>
+        </InternalParameters>
+
+   </Plugin>
+
+Because in our initial compartment arrangement we have two ``Top`` cells capping the "ends" of the cluster we want to allow only
+a single internal (i.e. between compartments) link between ``Top`` and ``Center`` cells. However for ``Center`` cells
+we will allow two internal links. Let's run the simulation and turn on visualization of the links (Visualization-> FPP Links):
+
+After few MCS (FPP links might take few MCS to form because there is stochasticity involved in establishing links between cells)
+we will see the following picture
+
+|img010|
+
+If we let the simulation run for a while, however, we will see that while the distance between cells is maintained the cells do not
+form "elongated cell"
+
+|img011|
+
+Additionally, if we lower FPP Lambdas from 100 to 10:
+
+.. code-block:: XML
+
+    <Plugin Name="FocalPointPlasticity">
+
+        <InternalParameters Type1="Top" Type2="Center">
+            <Lambda>10.0</Lambda>
+            <ActivationEnergy>-50.0</ActivationEnergy>
+            <TargetDistance>5</TargetDistance>
+            <MaxDistance>10.0</MaxDistance>
+            <MaxNumberOfJunctions>1</MaxNumberOfJunctions>
+        </InternalParameters>
+
+        <InternalParameters Type1="Center" Type2="Center">
+            <Lambda>10.0</Lambda>
+            <ActivationEnergy>-50.0</ActivationEnergy>
+            <TargetDistance>5</TargetDistance>
+            <MaxDistance>10.0</MaxDistance>
+            <MaxNumberOfJunctions>2</MaxNumberOfJunctions>
+        </InternalParameters>
+
+   </Plugin>
+
+
+We will see that ``Center`` cells that initially touched ``Top``  cell form additional links between themselves.
+This happens because those two ``Center`` cells can form two links between Center themselves. The first link is formed
+at the beginning of the simulation but during the course of the simulation , when those two ``Center`` cells come together
+(e.g. due to weak FPP Lambda) there is nothing keeping them form forming another link.
+
+|img012|
+
+.. note::
+
+    CC3D will add additional constraint on the max number of links a given cell type can form which will solve this problem
+    but in the current version we have to deal possible issues that might arise when cells may form extra link that we do not want
+
+
+Curvature Plugin
+++++++++++++++++
+
+Let us now put everything together an implement elongated compartmentalized cell. The solution that will prevent two
+``Center`` cells (the ones that initially were touching ``Top`` cell), from forming an extra FPP link,
+is to use Curvature Plugin. The way Curvature plugin works is by constraining the angle that two adjacent links can form
```

Comment:
Change sentence to: "The Curvature plugin constrains the angle at which two adjacent links can form."